week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,59 @@
 // Write your code here
+const imageArray = [
```

Comment:
Nice solution, well done! It works well generally but I think there are a few little bugs to do with how you are looping through the images.

You should manipulate the i first, and only set the image at the end. So something like:
```
  i++;
  if (i >= imageArray.length) {
    i = 0;
  }
  getImages.src = imageArray[i];
```